src/lib/core/components/computations/plugins/alphaDiv.tsx

```diff
@@ -1,25 +1,86 @@
 /** @jsxImportSource @emotion/react */
-import { jsx } from '@emotion/react';
-import { useRouteMatch } from 'react-router-dom';
-import { useHistory } from 'react-router';
 import { useStudyMetadata } from '../../..';
+import { StudyEntity } from '../../../types/study';
 import { useCollectionVariables } from '../../../hooks/study';
 import { VariableDescriptor } from '../../../types/variable';
 import { boxplotVisualization } from '../../visualizations/implementations/BoxplotVisualization';
 import { scatterplotVisualization } from '../../visualizations/implementations/ScatterplotVisualization';
 import { ComputationConfigProps, ComputationPlugin } from '../Types';
 import { H6 } from '@veupathdb/coreui';
 import { isEqual } from 'lodash';
-import { createComputation } from '../Utils';
+import { useConfigChangeHandler, assertComputationWithConfig } from '../Utils';
+import { findCollections } from '../../../utils/study-metadata';
+import * as t from 'io-ts';
+import { Computation } from '../../../types/visualization';
+
+export type AlphaDivConfig = t.TypeOf<typeof AlphaDivConfig>;
+// eslint-disable-next-line @typescript-eslint/no-redeclare
+export const AlphaDivConfig = t.type({
+  name: t.string,
+  collectionVariable: VariableDescriptor,
+  alphaDivMethod: t.string,
+});
 
 export const plugin: ComputationPlugin = {
   configurationComponent: AlphaDivConfiguration,
+  configurationDescriptionComponent: AlphaDivConfigDescriptionComponent,
   visualizationTypes: {
     boxplot: boxplotVisualization,
     scatterplot: scatterplotVisualization,
   },
+  createDefaultComputationSpec: createDefaultComputationSpec,
 };
 
+function AlphaDivConfigDescriptionComponent({
+  computation,
+}: {
+  computation: Computation;
+}) {
+  const studyMetadata = useStudyMetadata();
+  const collections = useCollectionVariables(studyMetadata.rootEntity);
+  assertComputationWithConfig<AlphaDivConfig>(computation, Computation);
+  const { configuration } = computation.descriptor;
+  const updatedCollectionVariable = collections.find((collectionVar) =>
+    isEqual(
+      {
+        variableId: collectionVar.id,
+        entityId: collectionVar.entityId,
+      },
+      configuration.collectionVariable
+    )
+  );
+  return (
+    <>
+      <h4 style={{ padding: '15px 0 0 0', marginLeft: 20 }}>
+        Data:{' '}
+        <span style={{ fontWeight: 300 }}>
+          {`${updatedCollectionVariable?.entityDisplayName} > ${updatedCollectionVariable?.displayName}`}
+        </span>
+      </h4>
+      <h4 style={{ padding: 0, marginLeft: 20 }}>
+        Method:{' '}
+        <span style={{ fontWeight: 300 }}>
+          {configuration.alphaDivMethod[0].toUpperCase() +
+            configuration.alphaDivMethod.slice(1)}
+        </span>
+      </h4>
+    </>
+  );
+}
+
+function createDefaultComputationSpec(rootEntity: StudyEntity) {
```

Comment:
also really nice. Will make it easy to add more complex logic when picking default configurations